typescript/infra/scripts/check-deploy.ts

```diff
@@ -111,6 +115,42 @@ async function check() {
       ismFactory,
     );
     governor = new ProxiedRouterGovernor(checker);
+  } else if (module === Modules.WARP) {
+    // test config
+    const plumetestnet = {
+      ...routerConfig.plumetestnet,
+      type: TokenType.synthetic,
+      name: 'Wrapped Ether',
+      symbol: 'WETH',
+      decimals: 18,
+      totalSupply: '0',
+      gas: 0,
+    };
```

Comment:
nbd but should the script take this info as params or something? Or at least have the consts defined at the top

Reply:
Ya, that would be better. I was just mimicking the deployer setup.